### ISSUE TITLE
TypeError: an integer is required

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#44]
```
Traceback (most recent call last):

> File "/usr/bin/gphotoframe", line 22, in <module>
> PhotoListStore()
> File "/usr/lib/python2.6/site-packages/gphotoframe/photoliststore.py",
line 25, in __init__
> self.photoframe = PhotoFrameFactory().create(self)
> File "/usr/lib/python2.6/site-packages/gphotoframe/photoframe.py", line
23, in create
> photoframe = PhotoFrame(photolist)
> File "/usr/lib/python2.6/site-packages/gphotoframe/photoframe.py", line
46, in __init__
> self._set_window_position()
> File "/usr/lib/python2.6/site-packages/gphotoframe/photoframe.py", line
78, in _set_window_position
> self.conf.get_int('root_y'))
> TypeError: an integer is required

```

Original issue reported on code.google.com by `yendo0206` on 2 Mar 2010 at 5:14

**Comment by GoogleCodeExporter** [comment]
```
It is a bug. But, you need register the gconf schemas, if you installed 
gphotoframe.

  % sudo gconf-schemas --register gphotoframe.schemas

Or, you can set the gconf data to avoid this error:

  % gconftool-2 --type int --set /apps/gphotoframe/root_x 0
  % gconftool-2 --type int --set /apps/gphotoframe/root_y 0

```

Original comment by `yendo0206` on 2 Mar 2010 at 5:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revison 634e448fb2.  
```

Original comment by `yendo0206` on 2 Mar 2010 at 6:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Many thanks Gphotoframe work now.
```

Original comment by `gohan...@gmail.com` on 2 Mar 2010 at 1:20
